docs/sources/reference/components/faro/faro.receiver.md

```diff
@@ -115,18 +115,31 @@ If the `api_key` argument is empty, no authentication checks are performed, and
 
 The `rate_limiting` block configures rate limiting for client requests.
 
-| Name         | Type     | Description                          | Default | Required |
-|--------------|----------|--------------------------------------|---------|----------|
-| `enabled`    | `bool`   | Whether to enable rate limiting.     | `true`  | no       |
-| `rate`       | `number` | Rate of allowed requests per second. | `50`    | no       |
-| `burst_size` | `number` | Allowed burst size of requests.      | `100`   | no       |
+| Name                 | Type     | Description                                          | Default | Required |
+| -------------------- | -------- | ---------------------------------------------------- | ------- | -------- |
+| `enabled`            | `bool`   | Whether to enable rate limiting.                     | `true`  | no       |
+| `rate`               | `number` | Rate of allowed requests per second.                 | `50`    | no       |
+| `burst_size`         | `number` | Allowed burst size of requests.                      | `100`   | no       |
+| `per_app_enabled`    | `bool`   | Whether to enable per-application rate limiting.     | `false` | no       |
+| `per_app_rate`       | `number` | Rate of allowed requests per second per application. | `50`    | no       |
+| `per_app_burst_size` | `number` | Allowed burst size of requests per application.      | `100`   | no       |
```

Comment:
WDYT about retaining the original 3 attributes and make only one new attribute called `strategy` that could default to `global`, but it could also be set to `per_app`? It wouldn't make sense to have both types of limit enabled, and it looks like the implementation in the PR will just ignore the global limits if the per app limits are enabled?

I also wonder if this could be made even more configurable by configuring which part of the json payload we could load balance by, so that users can choose something other than app name. But maybe that's overkill and not needed in practice?